kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMConfigurationServiceImpl.java

```diff
@@ -389,11 +389,6 @@ private void writeDhcpServerConfiguration(Set<String> interfaceNames) {
         });
     }
 
-    protected DhcpServerConfigWriter buildDhcpServerConfigWriter(final String interfaceName,
```

Comment:
I think that this method is used only for testing. Indeed the tests are failing.